aptos-move/framework/aptos-framework/sources/chain_id.move

```diff
@@ -21,4 +21,18 @@ module aptos_framework::chain_id {
     public fun get(): u8 acquires ChainId {
         borrow_global<ChainId>(@aptos_framework).id
     }
+
+    #[test_only]
+    fun change_id(value: u8) acquires ChainId {
```

Comment:
This should be public or you won't be able to access this anyway :)

Reply:
Sorry, I was careless.